prometheus-exporters-formula/prometheus-exporters/map.jinja

```diff
@@ -33,4 +36,3 @@
         'postgres_exporter_service_config': '/etc/sysconfig/prometheus-postgres_exporter',
     },
 }, merge=salt['pillar.get']('exporters:lookup')) %}
-
```

Comment:
As far as I understand git usually likes to have a newline at the end of each file, so you might want to consider not removing it here.